samples/cpp/model_creation_sample/main.cpp

```diff
@@ -77,10 +77,13 @@ ov::Tensor read_weights(const std::string& filepath) {
  */
 std::shared_ptr<ov::Model> create_model(const std::string& path_to_weights) {
     const ov::Tensor weights = read_weights(path_to_weights);
-    const std::uint8_t* data = weights.data<std::uint8_t>();
+    OPENVINO_SUPPRESS_DEPRECATED_START  // keep until 2026.0 release
+        const std::uint8_t* data = weights.data<std::uint8_t>();
+    OPENVINO_SUPPRESS_DEPRECATED_END  // keep until 2026.0 release
 
-    // -------input------
-    std::vector<ptrdiff_t> padBegin{0, 0};
+        // -------input------
```

Comment:
please, fix indentation

Reply:
It was done by formatter.
It looks like when there is a comment after `OPENVINO_SUPPRESS_DEPRECATED_END  // keep until 2026.0 release`, it gives such result. 
Comment removed.